### PR TITLE
Symbol Matching

### PR DESCRIPTION
**Pull request by hohle**
Matching several symbols for consistent names across duplicates.

### CODE REVIEW

**Comment by hohle** [comment]
While working on other dups, I was running into symbols that were known in other objects and did a quick pass through `duplicates.txt` to find known names.

**Comment by hohle** [comment]
Looking into build failures.

**Comment by joshlory** [comment]
Nice cleanup!

File: src/st/no3/44C64.c [hunk 1]
```diff
@@ -255,16 +255,20 @@ u16 GetNormalizedAngle(u16 arg0, u16 arg1, u16 arg2) {
 }
 
 void SetStep(u8 step) {
```
Comment (sozud) [inline src/st/no3/44C64.c:257]: Do the psp versions of these not match everywhere?

Comment (hohle) [reply src/st/no3/44C64.c:257]: Yes, the PSP version should be the default. Fixed.

Comment (sozud) [reply src/st/no3/44C64.c:257]: It's still not the psp version?

**Review by Xeeynamo — APPROVED** [review]
Looks good!

For `SetStep` it is fine as it is. We will split that portion of code into `st_common.c` and normalize it as we're doing with the rest of the code anyway

**Pull request merged**